### PR TITLE
Inform students of their bee's next active period

### PR DESCRIPTION
**Pull request by UtkarshKr007**
Closes: #210

### CODE REVIEW

File: src/app/services/student-round.service.ts [hunk 1]
```diff
@@ -145,6 +147,21 @@ export class StudentRoundService {
...
+  * in relation to the current time of the the round.
+  * - `null` if bee doesn't have anymore active periods in the current round
+  */
+  nextActivePeriod$: Observable<TimePeriod | null> = combineLatest(
+    [this.currentBeeSpecies$, this.currentTime$]
+  ).pipe(
+    filter(([species, time]) => !!species && !!time),
+    map(([species, time]) =>
+        species.active_period.find(interval => time.time < interval[0].time)?.[0] ?? null
+    ),
```
Comment (floogulinc) [inline src/app/services/student-round.service.ts:161]:
```suggestion
    map(([species, time]) =>
      species.active_period.find(interval => time.time < interval[0].time)?.[0] ?? null
    ),
```

**Review by floogulinc — DISMISSED** [review]
Just one minor formatting thing but otherwise looks great!

**Review by floogulinc — APPROVED** [review]

File: src/app/components/student-round/student-round.component.html [hunk 2]
```diff
@@ -4,14 +4,25 @@
...
       <mat-card *ngSwitchCase="ScreenId.InactiveBee">
-        <mat-card-title>Your {{(roundService.currentBeeSpecies$ | async).name}} is currently inactive</mat-card-title>
+        <img
+          class="student-bee"
+          *ngIf="(roundService.currentBeeSpecies$ | async) as bee"
+          src="/assets/art/500w/bees/{{bee.art_file}}"
+          [alt]="bee.name"
+        >
+        <mat-card-title>
+          {{(roundService.currentBeeSpecies$ | async).name | sentenceCase}} are currently inactive
```
Comment (floogulinc) [inline src/app/components/student-round/student-round.component.html:21]:
```suggestion
          {{(roundService.currentBeeSpecies$ | async).name | sentenceCase}}s are not active right now
```

Joe, KK, and I discussed this phrasing and we want to change it a little bit and pluralize the bee name (they all end in "bee" right now so we should make it "bees").

File: src/app/components/student-round/student-round.component.html [hunk 3]
```diff
@@ -4,14 +4,25 @@
...
+        </mat-card-title>
+        <mat-card-subtitle *ngIf="(nextActivePeriodText$| async) as nextActivePeriodText">
+          You will next be active in {{nextActivePeriodText}}
```
Comment (floogulinc) [inline src/app/components/student-round/student-round.component.html:24]:
```suggestion
          You will next be active {{nextActivePeriodText}}
```

The "in" is included in this string where needed. We don't want it to say "in soon" for example.

**Review by floogulinc — CHANGES_REQUESTED** [review]
Joe, KK, and I discussed some of the phrasing and text stuff and have a suggested change to the phrasing of the part that says their species is not active. I also caught a little issue that I think happened with all the branch merging and stuff that happened.

**Review by floogulinc — APPROVED** [review]

**Review by kklamberty — APPROVED** [review]
This looks great! Nice work!

**Pull request merged**